framework/src/CMakeLists.txt

```diff
@@ -43,6 +43,10 @@ if("${DF_TARGET}" STREQUAL "nuttx")
 		)
 	add_dependencies(df_driver_framework prebuild_targets)
 else()
+	include_directories(../../../Robotics_Cape_Installer/library/include)
```

Comment:
Is this needed? And if so, can we move it to Toolchain-bbblue.cmake?

Reply:
No. I missed it last time. Removed.